python/lsst/ci/build.py

```diff
@@ -46,10 +47,19 @@ def declare_eups_tag(tag, eups_obj):
 
 
 class ProgressReporter:
-    # progress reporter: display the version string as progress bar, character by character
+    """Class that that displays the version string as a progress bar as an
+    indicator of liveness.
+
+    Parameters
+    ----------
+    out
+        file this class will write the progress to
+    product
+        name of the product to indicate progress on
```

Comment:
Is this really a name (str)? It has type models.Product not str, which seems surprising.

Reply:
it's actually a models.Product. docstring has been updated.